Add reference docs for The Pop-Up API, and my own author details (#4069)
GoogleChrome/developer.chrome.com: site/en/docs/web-platform/popup-api/open-pseudo-class/index.md

---
layout: 'layouts/doc-post.njk'
title: :open
description: Allows a pop-up to be styled, but only when it is being shown.
subhead: Allows a pop-up to be styled, but only when it is being shown.
authors:
  - chrisdavidmills
date: 2022-10-19
updated: 2022-10-19
---

The **`:open`** [CSS](https://developer.mozilla.org/docs/Web/CSS) [pseudo-class](https://developer.mozilla.org/docs/Web/CSS/Pseudo-classes) allows a [pop-up](/docs/web-platform/popup-api/) to be styled, but only when it is being shown.

## Syntax

```css
:open
```

## Example

> In this case we are transitioning the pop-up in from the bottom of the viewport, rather than just having it appear in the center:

Reviewer: ```suggestion
The following example transitions the pop-up in from the bottom of the viewport, rather than have it appear in the center:
```
